src/lib/Repository/UiUserService.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\HybridPlatformUi\Repository;
+
+use eZ\Publish\API\Repository\Exceptions\NotFoundException;
+use eZ\Publish\API\Repository\Repository;
+use eZ\Publish\API\Repository\UserService;
+
+/**
+ * Service for loading user.
+ */
+class UiUserService
+{
+    /**
+     * @var UserService
+     */
+    private $userService;
+
+    /**
+     * @var Repository
+     */
+    private $repository;
+
+    public function __construct(Repository $repository, UserService $userService)
+    {
+        $this->repository = $repository;
+        $this->userService = $userService;
+    }
+
+    /**
+     * Loads a user ignoring privileges.
+     *
+     * @param mixed $userId
+     *
+     * @return \eZ\Publish\API\Repository\Values\User\User
+     */
+    public function loadUserIgnoringPermissions($userId)
+    {
+        return $this->repository->sudo(function () use ($userId) {
+            return $this->loadUser($userId);
+        });
+    }
+
+    /**
+     * Loads user and handles not found exception.
+     *
+     * @param mixed $userId
+     *
+     * @return \eZ\Publish\API\Repository\Values\User\User|null
+     */
+    public function loadUser($userId)
+    {
+        try {
+            return $this->userService->loadUser($userId);
+        } catch (NotFoundException $e) {
+            return null;
```

Comment:
These got mixed up a bit.

Our API, which this software is based upon, throws NotFound exceptions when a `load()` method is called, and doesn't find the requested item. We have our own standards. We call the ones who may not find what they're asked for `findSomething()`.

Since we have `findUserById()` above, and `loadUser()` is not used anymore, we can just remove it, and ignore the method altogether.